### PR TITLE
Doc update mongo postgres separate listeners

### PR DESCRIPTION
**Pull request by smallinsky**
## What:

Docs update for Mongo Postgres separate listeners.

### CODE REVIEW

**Review by klizhentas — APPROVED** [review]

**Comment by klizhentas** [comment]
@smallinsky please merge

**Review by jakule — APPROVED** [review]

File: docs/pages/database-access/reference/configuration.mdx [hunk 1]
```diff
@@ -85,14 +85,22 @@ proxy_service:
   # MySQL proxy is listening on a separate port and needs to be enabled
   # on the proxy server.
   mysql_listen_addr: "0.0.0.0:3036"
+  # Postgres proxy is listening address. If provided proxy will use a separate listener
```
Comment (r0mant) [inline docs/pages/database-access/reference/configuration.mdx:88]:
```suggestion
  # Postgres proxy listening address. If provided, proxy will use a separate listener
```

File: docs/pages/database-access/reference/configuration.mdx [hunk 2]
```diff
@@ -85,14 +85,22 @@ proxy_service:
...
+  # instead of multiplexing Postgres protocol on web_listener_addr.
+  postgres_listen_addr: "0.0.0.0:5432"
+  # Mongo proxy is listening address. If provided proxy will use a separate listener
```
Comment (r0mant) [inline docs/pages/database-access/reference/configuration.mdx:91]:
```suggestion
  # Mongo proxy listening address. If provided, proxy will use a separate listener
```

File: docs/pages/database-access/reference/configuration.mdx [hunk 3]
```diff
@@ -85,14 +85,22 @@ proxy_service:
   # MySQL proxy is listening on a separate port and needs to be enabled
   # on the proxy server.
   mysql_listen_addr: "0.0.0.0:3036"
+  # Postgres proxy is listening address. If provided proxy will use a separate listener
+  # instead of multiplexing Postgres protocol on web_listener_addr.
+  postgres_listen_addr: "0.0.0.0:5432"
```
Comment (r0mant) [inline docs/pages/database-access/reference/configuration.mdx:90]: Let's comment out these to indicate they're optional and by default web listener is used.
```suggestion
  # postgres_listen_addr: "0.0.0.0:5432"
```

**Review by r0mant — APPROVED** [review]
lgtm after a couple of tweaks

**Pull request merged**